src/Data/Aeson/TypeScript/Types.hs

```diff
@@ -91,20 +91,29 @@ instance IsString (TSString a) where
 
 -- * Formatting options
 
+data ExportMode =
+  ExportEach
+  -- ^ Prefix every declaration with the "export" keyword (suitable for putting in a TypeScripe module)
+  | ExportNone
+  -- ^ No exporting (suitable for putting in a .d.ts file)
+
 data FormattingOptions = FormattingOptions
   { numIndentSpaces       :: Int
   -- ^ How many spaces to indent TypeScript blocks
   , interfaceNameModifier :: String -> String
   -- ^ Function applied to generated interface names
   , typeNameModifier :: String -> String
   -- ^ Function applied to generated type names
+  , exportTypes :: ExportMode
```

Comment:
Thanks, could you just rename this to `exportMode`? Otherwise LGTM.

Reply:
my bad!